index.html

```diff
@@ -46,6 +47,191 @@ <h1 class="margin">I'm Gabriel. <br> Glad to see <br> you!</h1>
+                </li>
+            </ul>
+            <p class="card-item">A daily selection of privately personalized reads; no accounts or sign-ups required.
+            </p>
+            <ul class="categories card-item">
+                <li class="html">
+                    <p>html</p>
+                </li>
+                <li>
+                    <p>css</p>
+                </li>
+                <li>
+                    <p>javascript</p>
+                </li>
+            </ul>
+            <div class="div-button"><button class="card-item">See project</button></div>
+        </div>
+        <div class="card">
+            <img class="card-img card-item" src="images/Snapshoot Portfolio 2.png" alt="">
+            <h3 class="card-item">
+                Multi-Post Stories
+            </h3>
+            <ul class="card-item company">
+                <li class="canopy">
+                    <h4>CANOPY</h4>
+                </li>
+                <li><img src="images/Counter.png"></li>
+                <li>
+                    <p>Back End Dev</p>
+                </li>
+                <li><img src="images/Counter.png"></li>
+                <li>
+                    <p>2015</p>
+                </li>
+            </ul>
+            <p class="card-item">A daily selection of privately personalized reads; no accounts or sign-ups required.
+            </p>
+            <ul class="categories card-item">
+                <li class="html">
+                    <p>html</p>
+                </li>
+                <li>
+                    <p>css</p>
+                </li>
+                <li>
+                    <p>javascript</p>
+                </li>
+            </ul>
+            <div class="div-button"><button class="card-item">See project</button></div>
+        </div>
+        <div class="card">
+            <img class="card-img card-item" src="images/Snapshoot Portfolio 3.png" alt="">
+            <h3 class="card-item">
+                Tonic
+            </h3>
+            <ul class="card-item company">
+                <li class="canopy">
+                    <h4>CANOPY</h4>
+                </li>
+                <li><img src="images/Counter.png"></li>
+                <li>
+                    <p>Back End Dev</p>
+                </li>
+                <li><img src="images/Counter.png"></li>
+                <li>
+                    <p>2015</p>
+                </li>
+            </ul>
+            <p class="card-item">A daily selection of privately personalized reads; no accounts or sign-ups required.
+            </p>
+            <ul class="categories card-item">
+                <li class="html">
+                    <p>html</p>
+                </li>
+                <li>
+                    <p>css</p>
+                </li>
+                <li>
+                    <p>javascript</p>
+                </li>
+            </ul>
+            <div class="div-button"><button class="card-item">See project</button></div>
```

Comment:
- [ ] Kindly ensure that your buttons also follow the design guidelines, such as `enabled`, `hover`, `pressed`, and `disabled`, so as to have a pixel-perfect design like you'll soon be doing in your job 🙂.

- Your design
![mobile](https://user-images.githubusercontent.com/41163571/235327622-029224c0-6b91-402e-8a24-3be1fb2d50cf.gif)

- Figma design
<img width="446" alt="right" src="https://user-images.githubusercontent.com/41163571/227707940-ae12a5de-15d8-4b79-b06b-a16bfdafd017.png">

- For all the buttons.

Reply:
Got it! I'm going to add the interactions to the buttons.

Reply:
Done! ✔
I implemented button interactions acroding to the figma desing.